docs/auth.md

```diff
@@ -0,0 +1,73 @@
+# Authorization Guide
+
+## Overview
+
+Optimizely Agent supports authorization workflows based on OAuth and JWT standards, allowing you to protect access to its API and Admin interfaces.
+
+There are three modes of operation:
+
+### Issuer & Validator
+Access tokens are issued by Agent itself, using a [Client Credentials grant](https://www.oauth.com/oauth2-servers/access-tokens/client-credentials/). Access tokens are signed and validated using the HS256 algorithm with a signing secret provided in configuration.
+
+### Validator-only
```

Comment:
If possible, it would be great to add one line to each option explaining why someone would choose one option over the other.